src/Components/Header.js

```diff
@@ -40,6 +40,7 @@ export default function Header(props) {
                 color: "#959595",
                 fontSize: "14.74px",
                 fontFamily: "Roboto",
+                paddingTop: "2px",
```

Comment:
I think 3 px will be nice @ArslanYM 
```suggestion
                paddingTop: "3px",
```